datadog-trace-utils/src/send_with_retry/mod.rs

```diff
@@ -223,41 +223,16 @@ pub async fn send_with_retry(
 }
 
 async fn send_request(
+    client: &HttpClient,
     timeout: Duration,
     req: HttpRequestBuilder,
     payload: Bytes,
-    http_proxy: Option<&str>,
 ) -> Result<hyper_migration::HttpResponse, RequestError> {
     let req = req
         .body(hyper_migration::Body::from_bytes(payload))
         .or(Err(RequestError::Build))?;
 
-    #[cfg(feature = "proxy")]
```

Comment:
Who was the user of this `proxy` feature? Is the refactor cleared with them?

Reply:
The proxy feature was introduced by the serverless team. I shared the the PR when it was a draft with AJ and the response was positive. He was going to run some tests, I'm waiting for his feedback before merging.

Reply:
Sounds like s plan